### PR TITLE
Add missing import

### PR DESCRIPTION
**Pull request by gcapalbo**
@snopoke @czue

### CODE REVIEW

File: custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {% extends "pact/patient/pactpatient_base.html" %}
 {% load case_tags %}
+{% load hq_shared_tags %}
+
 {% block patient-tab-container %}
+<script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```
Comment (gcapalbo) [inline custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html]: @dimagi/js-team it looks like the timeago import used to be in bootstrap2/base.html but it's not in boostrap3/base.html anymore. should it be, or should we just import it here directly? i think this issue was introduced when this report was migrated to bootstrap 3, and this report will face the same problem when it's upgraded: https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/reports/templates/reports/careplan/patient_careplan.html

Comment (orangejenny) [inline custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html]: Given that this library isn't used often, putting it in the reports themselves sounds preferable to me.

Probably better to do it now (remove it from B2 base and add it to the reports that need it) than to remember as each report is upgraded.

Comment (gcapalbo) [inline custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html]: actually i just looked a little closer and it looks like the bootstrap2 template actually initializes timeago for all elements with the timeago class: https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/style/templates/style/bootstrap2/base.html#L360

i'm  thinking i'll just add the same to the boostrap3 base template unless there's any objections - i'm not sure how to reliably find out all the pages that need this.

**Comment by czue** [comment]
this is :+1: from my side as a short term fix. @gcapalbo let me know if you want me to merge or fell free to merge yourself if you want to get this out before figuring out whether it should go elsewhere.

**Comment by gcapalbo** [comment]
@czue thanks, i think it's fine to wait a few days to confirm (it's been like this for a couple weeks and i don't think anyone seemed to notice)

@dimagi/js-team i'm going to add the timeago import and initialization to the boostrap3/base.html template as it was for the bootstrap2/base.html template: https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/style/templates/style/bootstrap2/base.html#L360 . let me know by eod monday if you have any issues with that

**Comment by dannyroberts** [comment]
@gcapalbo this PR looks good to me. Any reason not to merge? (Feel free to self-merge also.)

File: corehq/apps/style/templates/style/bootstrap3/base.html [hunk 2]
```diff
@@ -508,6 +508,16 @@
         {% endcompress %}
         {% endif %}
 
+        {% if request.use_timeago %}
+        <script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```
Comment (gcapalbo) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: wasn't sure if this should be between the compress tag like the others are?

File: corehq/apps/style/templates/style/bootstrap3/base.html [hunk 3]
```diff
@@ -508,6 +508,16 @@
...
+        <script>
+            $(function () {
+                $.timeago.settings.allowFuture = true;
+                $(".timeago").timeago();
+            });
+        </script>
```
Comment (gcapalbo) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: (copied from https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/style/templates/style/bootstrap2/base.html#L357)

**Comment by gcapalbo** [comment]
this should be good to go now

File: corehq/apps/style/templates/style/bootstrap3/base.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/style/templates/style/bootstrap3/base.html]: the `if` statement can't be in the compress tag, so probably good to leave it here

**Comment by benrudolph** [comment]
sweet looks good to me, going to merge

**Pull request merged**